File: Ejercicios Laragon/proyectopython2644590/notes.py
notas = []

while len(notas) < 3:
    print(f"Write the data {len(notas)+1}")
    numero = float(input("Input a number: "))
    if numero >= 0 and numero <= 5:
        notas.append(numero)
    else:
        print("Input a valid data")

""" for num in range(1,4):
    numero = float(input("Ingrese un numero: "))
    notas.append(numero) """

suma = 0
for nota in notas:
    suma += nota

promedio = suma/3

print(f"Your average is {promedio}")

if promedio >= 3.5:
    print("Approved")
else:
    print("Failed")

File: Ejercicios Laragon/Activity/notes.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Promedio Notas</title>
</head>
<body>
    <form action="#" method="POST">
        <input type="number" name="nota1" id="nota1" placeholder="Ingrese la nota 1" max="5" min="0" step="any" required>
        <input type="number" name="nota2" id="nota2" placeholder="Ingrese la nota 2" max="5" min="0" step="any" required>
        <input type="number" name="nota3" id="nota3" placeholder="Ingrese la nota 3" max="5" min="0" step="any" required>
        <button type="submit">Promedio</button>
    </form>

    <?php
        if(isset($_POST['nota1']) && isset($_POST['nota2']) && isset($_POST['nota3'])){
            $nota1 = $_POST['nota1'];
            $nota2 = $_POST['nota2'];
            $nota3 = $_POST['nota3'];

            $suma = $nota1 + $nota2 + $nota3;

            $promedio = $suma/3;
            echo "Su promedio es $promedio"."<br>";

            if($promedio < 3.5){
                $message = "Has reprobado";
            }else{
                $message = "Has Aprobado";
            }

            echo $message;
        }
    ?>

</body>
</html>